CHANGES.md

```diff
@@ -120,6 +120,8 @@
 * If a main session fails to load, the pipeline will now fail at worker startup. ([#25401](https://github.com/apache/beam/issues/25401)).
 * Python pipeline options will now ignore unparsed command line flags prefixed with a single dash. ([#25943](https://github.com/apache/beam/issues/25943)).
 * The SmallestPerKey combiner now requires keyword-only arguments for specifying optional parameters, such as `key` and `reverse`. ([#25888](https://github.com/apache/beam/issues/25888)).
+* The google-cloud-profiler dependency was accidentally removed from Beam's Python Docker
```

Comment:
(minor) does this better fix "known issues" section?

Reply:
good call.